sources/models/README.md

```diff
@@ -57,7 +57,7 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 * [Model](src/aws-dev/mod.rs)
```

Comment:
Also a number of `VMWare`s in `sources/api/early-boot-config/src/provider/vmware.rs`